modules/reference/pages/cluster-properties.adoc

```diff
@@ -1,1796 +1,5372 @@
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `524288`
+
+---
+
+=== raft_recovery_throttle_disable_dynamic_mode
+
+Disables dynamic rate allocation in recovery throttle (advanced).
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* boolean
+
+*Default:* `false`
+
+---
+
+=== raft_replica_max_pending_flush_bytes
+
+Max not flushed bytes per partition. If configured threshold is reached log will automatically be flushed even though it wasn't explicitly requested.
+
+*Requires Restart:* No
+
+*Nullable:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `262144`
+
+---
+
+=== raft_replicate_batch_window_size
+
+Max size of requests cached for replication.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `1048576`
+
+---
+
+=== raft_smp_max_non_local_requests
+
+Maximum number of x-core requests pending in Raft seastar::smp group. (for more details look at `seastar::smp_service_group` documentation).
+
+*Requires Restart:* No
+
+*Nullable:* Yes
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`0`, `4294967295`]
+
+*Default:* `null`
+
+---
+
+=== raft_timeout_now_timeout_ms
+
+Timeout for a timeout now request.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `1s`
+
+---
+
+=== raft_transfer_leader_recovery_timeout_ms
+
+Timeout waiting for follower recovery when transferring leadership.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `10s`
+
+---
+
+=== readers_cache_eviction_timeout_ms
+
+Duration after which inactive readers will be evicted from cache.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `30s`
+
+---
+
+=== reclaim_batch_cache_min_free
+
+Free memory limit that will be kept by batch cache background reclaimer.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `67108864`
+
+---
+
+=== reclaim_growth_window
+
+Length of time in which reclaim sizes grow.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Accepted values:* [`-17592186044416`, `17592186044415`]
+
+*Default:* `3000ms`
+
+---
+
+=== reclaim_max_size
+
+Maximum batch cache reclaim size.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `4194304`
+
+---
+
+=== reclaim_min_size
+
+Minimum batch cache reclaim size.
+
+*Requires Restart:* No
+
+*Nullable:* No
+
+*Visibility:* `tunable`
+
+*Type:* integer
+
+*Default:* `131072`
+
+---
+
+=== reclaim_stable_window
+
+Length of time above which growth is reset.
```

Comment:
```suggestion
If the duration since the last time memory was reclaimed is longer than the amount of time specified in this property, the memory usage of the batch cache is considered stable, so only the minimum size (<<reclaim_min_size,reclaim_min_size>>) is set to be reclaimed.
```